build.gradle.kts

```diff
@@ -218,10 +218,6 @@ allprojects {
         }
 
         val checkstyleTasks = tasks.withType<Checkstyle>()
-        checkstyleTasks.configureEach {
-            // Checkstyle 8.26 does not need classpath, see https://github.com/gradle/gradle/issues/14227
-            classpath = files()
-        }
```

Comment:
As clarified in https://github.com/checkstyle/checkstyle/pull/12392#issuecomment-1313400070, I am inclined the removal of `classpath = files()` is not really needed.

Would you please revert this change and retest with a recent pgjdbc source code?

Reply:
A newer commit fails with checkstyle violations in your master branch from before this PR. This is the primary reason we were stuck on an old commit as we need a stable version to use in our CI.

Even if this was from an older gradle and gradle removed theirs, I am seeing this PR as a good change as the classpath is no longer needed even in your own code and leaving this here doesn't make sense as long the build passes. You won't be able to send the classpath to checkstyle.

Reply:
@rnveach , `classpath=files()` is REQUIRED to make the build faster.

Reply:
The comment refers https://github.com/gradle/gradle/issues/14227 which does contain exact explanation:

> checkstyleTests captures dependency on main classes via classpath, so :checkstyleTest is re-executed.

In other words, `checkstyleTest` task is re-executed even if user modifies "main" code only.

Please read https://github.com/gradle/gradle/issues/14227#issue-682593067 description.